EventListener/SecurityListener.php

```diff
@@ -65,7 +66,12 @@ public function onKernelController(FilterControllerEvent $event)
         }
 
         if (!$this->language->evaluate($configuration->getExpression(), $this->getVariables($request))) {
-            throw new AccessDeniedException(sprintf('Expression "%s" denied access.', $configuration->getExpression()));
+            if ($statusCode = $configuration->getStatusCode()) {
+                throw new HttpException($statusCode, $configuration->getMessage());
```

Comment:
Do you have a suggestion ?
I'm not sure `Expression "%s" denied access.` fits for all case